mlflow/genai/scorers/deepeval/__init__.py

```diff
@@ -94,13 +114,24 @@ def __call__(
         )
 
         try:
-            test_case = map_scorer_inputs_to_deepeval_test_case(
-                metric_name=self.name,
-                inputs=inputs,
-                outputs=outputs,
-                expectations=expectations,
-                trace=trace,
-            )
+            if self.is_session_level_scorer:
+                if session is None:
+                    raise MlflowException.invalid_parameter_value(
+                        f"Multi-turn metric '{self.name}' requires 'session' parameter "
```

Comment:
```suggestion
                        f"Multi-turn scorer '{self.name}' requires 'session' parameter "
```

I think we should unify to "scorer" (or maybe judge), at least when it is invoked via MLflow?